### PR TITLE
Update dep to bootstrap v4 beta 2

### PR DESCRIPTION
**Pull request by joneff**

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -119,6 +119,17 @@ $accent: #ff6358 !default;
...
 $accent-contrast: contrast-wcag( $accent ) !default;
 
+/// The color for informational messages and states.
+$info: #0058e9 !default;
+/// The color for success messages and states.
+$success: #37b400 !default;
+/// The color for warning messages and states.
+$warning: #ffc000 !default;
+/// The color for error messages and states.
+$error: #f31700 !default;
```
Comment (gyoshev) [inline scss/_variables.scss:129]: Why should we move these? :)

**Review by gyoshev — APPROVED** [review]
Looks good so far 👍  Not sure if anything else is necessary for merging this branch and the one in theme-bootstrap.

File: scss/_variables.scss (hunk 1 above)
Comment (joneff) [reply scss/_variables.scss:129]: I just wanted to group the variables together e.g. first main theme color and then the rest of the colors.

File: package.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
   "devDependencies": {
     "@telerik/kendo-common-tasks": "^3.0.0",
     "@telerik/semantic-prerelease": "^1.0.0",
-    "bootstrap": "git://github.com/twbs/bootstrap.git#95f37e4c402df37db16781995ffa1592032df9c8",
+    "bootstrap": "4.0.0-beta.2",
```
Comment (gyoshev) [inline package.json:71]: ❤️

**Pull request merged**